### ISSUE TITLE
Shift+arrow selection of text broken in Linux

### ISSUE THREAD

**Issue by zacjansheski** [musescore/MuseScore#29162]
### Issue type

UX/Interaction bug (incorrect behaviour)

Reference comment by @MarcSabatella 
https://github.com/musescore/MuseScore/issues/28916#issuecomment-3168802310

### Description with steps to reproduce

Issue is only on Linux, not an issue on Windows or Mac

1. Select text
2. Attempt to use shift+arrow to select text to right or left.  Stops after a single letter (or word with control modifier)

### Supporting files, videos and screenshots

See referenced comment

### What is the latest version of MuseScore Studio where this issue is present?

OS: Ubuntu 24.04.2 LTS, Arch.: x86_64, MuseScore Studio version (64-bit): 4.6.0-252200329, revision: github-musescore-musescore-11626d1

### Regression

Yes, this works in the latest released version and is broken in the nightly builds

### Operating system

Ubuntu 22.04.3.

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by ghost** [comment]
Actually same with dynamics popup:

- Select note
- Press Ctrl+D
- Dynamics popup appears; however, cannot type anything ('f', 'p', whatever) into the text field, nor does Shift work (if I enter a dynamic by clicking then want to move it about)
